### ISSUE TITLE
LOST AUTHENTICATOR CODES. SUDDEN RESET!

### ISSUE THREAD

**Issue by alchernartech243** [Authenticator-Extension/Authenticator#1199]
### Describe the issue

I opened autthenticator  code and suddenly it reset by its  own. Unfortunately there is no recovery or backup, Help!

### Browser

Chrome

### Browser Version

125.0.6422.142 (Official Build) (64-bit)

### Extension Version

7.0.0

**Comment by mymindstorm** [comment]
@Sneezry alongside #1197, do you think there might be a migration issue?

**Comment by Sneezry** [comment]
No, I don't think there is an account data migration issue. Account data is stored in `chrome.storage` already before the migration. And we only migrate setting data from `localStorage` to `chrome.storage` in MV3. Account data is untouched during the migration.

**Comment by Sneezry** [comment]
![image](https://github.com/Authenticator-Extension/Authenticator/assets/1621293/9431fbc9-6c4f-4b55-a001-b7e8208aac53)

We have rolled out version 7.0.0 to over 3.8 million Chrome users, and we haven't observed any significant increase in data loss reports.
